vm/src/sysmodule.rs

```diff
@@ -222,6 +222,78 @@ fn sys_displayhook(obj: PyObjectRef, vm: &VirtualMachine) -> PyResult<()> {
     Ok(())
 }
 
+#[pystruct_sequence(name = "sys.getwindowsversion")]
```

Comment:
Would you please rebase this PR to upstream? This macro have a bit different signature due to #2020.

Reply:
Would you mind if I rearrange commit history a little bit while rebasing? I'd like to clean up commits related to formatting.

Reply:
Cleaning up local history by rebasing is always recommended, welcome